src/Azure.Functions.Cli/Common/Constants.cs

```diff
@@ -233,5 +235,12 @@ public static class StaticResourcesNames
                         Name = "Microsoft.Azure.WebJobs.Extensions.Twilio" }
                     }
                 });
+
+        public enum HttpAuthorizationLevel
+        {
```

Comment:
Instead of defining another enum, is it possible to reuse : https://github.com/Azure/azure-webjobs-sdk-extensions/blob/dev/src/WebJobs.Extensions.Http/AuthorizationLevel.cs --> this will ensure we keep the code in sync